src/calculateRentalCost.js

```diff
@@ -29,6 +29,18 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+  const threeDaysDiscount = 20;
+  const sevenDaysDiscount = 50;
+  const summaryPrice = days * price;
+
+  if (days >= 7) {
+    return summaryPrice - sevenDaysDiscount;
+  } else if (days >= 3) {
+    return summaryPrice - threeDaysDiscount;
+  } else {
+    return summaryPrice;
+  }
 }
```

Comment:
Last else is redundant here
```suggestion
  
  return summaryPrice;
}
```